content/manual-references.json

```diff
@@ -18,5 +18,74 @@
       "family": "Himmelstein"
     }
   ]
-}
-]
\ No newline at end of file
+},
+{
```

Comment:
If the URL citation is working now, you can please revert this file to remove the manual reference.  Thanks.

Reply:
Sure thing, done now.